tests/unit/test_content_trimmed.py

```diff
@@ -47,3 +46,27 @@ def test_content_trimmed():
     expected_content = "{\"content\": \"WALTHAM, Mass.--(BUSINESS WIRE)-- Software intelligence company Dynatrace (N[TRUNCATED]"
     assert len(actual_output["content"]) == content_length_limit
     assert actual_output["content"] == expected_content
+
+
+def test_content_with_exact_len_not_trimmed():
+    message = "WALTHAM, Mass.--(BUSINESS WIRE)-- Software intelligence company Dynatrace (NYSE: DT)"
+    content_length_limit_backup = logs_ingest.main.content_length_limit
+
+    # given
+    log_entry = create_log_entry(message)
+    logs_ingest.main.content_length_limit = len(json.dumps(log_entry))
```

Comment:
I would do it like in previous test:
```
content_length_limit = 100
```
or
```
content_length_limit = len(message)
```
Right now we're also counting the JSON braces and field name into the length limit so it wouldn't be truncated whether we were overly eager or not.

Reply:
Nevermind, Ira explained to me how it works here